packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -152,92 +260,79 @@ export default function OrgNav({
-            </Tooltip>
+              <p className={accessManagerMenuItem}>
+                <strong>Organization Access:</strong> {current.orgName}
+              </p>
+              <p className={accessManagerMenuItem}>
+                <strong>Project Access:</strong> {currentProjectName ?? 'None'}
+              </p>
+            </Menu>
           </li>
           <li role="none" className={supportContainer}>
-            <Tooltip
-              align="bottom"
-              justify="middle"
-              variant="dark"
-              trigger={
-                <a
-                  href={orgNav.support}
-                  className={cx(linkText, {
-                    [activeLink]: activeNav === 'support',
-                  })}
-                >
-                  Support
-                </a>
-              }
+            <LinkElement
+              href={orgNav.support}
+              isActive={activeNav === 'support'}
+              tooltipText="View the Organization Support"
             >
-              Organization Support
-            </Tooltip>
+              Support
+            </LinkElement>
           </li>
           <li role="none">
-            <Tooltip
-              align="bottom"
-              justify="middle"
-              variant="dark"
-              trigger={
-                <a
-                  href={orgNav.billing}
-                  className={cx(linkText, {
-                    [activeLink]: activeNav === 'billing',
-                  })}
-                >
-                  Billing
-                </a>
-              }
+            <LinkElement
+              href={orgNav.billing}
+              isActive={activeNav === 'billing'}
+              tooltipText="View the Organization Billing"
             >
               Billing
-            </Tooltip>
+            </LinkElement>
           </li>
         </ul>
       </div>
       <div>
-        <Tooltip
-          align="bottom"
-          justify="middle"
-          variant="dark"
-          trigger={
-            <a
-              href={orgNav.allClusters}
-              className={cx(rightSideLinkStyle, linkText, {
-                [activeLink]: activeNav === 'allClusters',
-              })}
-            >
-              All Clusters
-            </a>
-          }
+        <LinkElement
+          href={orgNav.allClusters}
+          isActive={activeNav === 'allClusters'}
+          tooltipText="View all clusters across your organizations"
+          className={css`
+            margin-right: 25px;
+          `}
         >
-          View All Clusters
-        </Tooltip>
-
+          All Clusters
+        </LinkElement>
         {admin && (
           <a
             href={orgNav.admin}
             className={cx(rightSideLinkStyle, linkText, {
               [activeLink]: activeNav === 'admin',
+              [navItemFocusStyle]: showFocus,
             })}
           >
-            Admin
+            <span>Admin</span>
```

Comment:
Why the span?

Reply:
I think I was originally styling it but then moved that elsewhere, my bad!